Dockerfile-test

```diff
@@ -45,7 +45,7 @@ COPY --from=builder /tmp/jq-linux64 /usr/local/bin/jq
 COPY --from=builder /bin/bash /bin/bash
 COPY --from=builder /lib/x86_64-linux-gnu/libtinfo.so.5 /lib64/libtinfo.so.5
 
-
+USER root
```

Comment:
Fallback to `USER ipfs` after the `mkdir` and `chown` `RUN` command as there shouldn't be any need to be root after that.